booking_engine/data/base_automation.xml

```diff
@@ -15,4 +15,25 @@
     <field name="filter_domain">[('rental_start_date', '!=', False), ('rental_return_date', '!=', False)]</field>
     <field name="trigger_field_ids" eval="[(6, 0, [ref('sale_renting.field_sale_order__rental_start_date'), ref('sale_renting.field_sale_order__rental_return_date')])]"/>
   </record>
+  <record id="industry_on_product_template_create" model="base.automation">
+    <field name="model_id" ref="product.model_product_template"/>
+    <field name="action_server_ids" eval="[(6, 0, [ref('industry_create_roles')])]"/>
+    <field name="trigger">on_create</field>
+    <field name="filter_domain">[("x_is_a_stay_offer", "!=", False)]</field>
```

Comment:
why only on create? Could be done on edition as well if there is no linked role yet, right?